plutus-example/plutus-example.cabal

```diff
@@ -0,0 +1,155 @@
+                        PlutusExample.AlwaysSucceeds
+                        PlutusExample.CustomDatumRedeemerGuess
+                        PlutusExample.DatumRedeemerGuess
+                        PlutusExample.MintingScript
+                        PlutusExample.ScriptContextChecker
+                        PlutusExample.Sum
+                        PlutusExample.Loop
+
+  build-depends:        aeson
+                      , bytestring
+                      , cardano-api
+                      , cardano-cli
+                      , cardano-ledger-alonzo
+                      , cardano-ledger-core
+                      , cardano-slotting
+                      , containers
+                      , ouroboros-consensus
+                      , ouroboros-network
+                      , plutus-ledger
+                      , plutus-ledger-api
+                      , plutus-tx
+                      , plutus-tx-plugin
+                      , serialise
+                      , strict-containers
+                      , transformers
+                      , transformers-except
+
+
+executable plutus-example
+  import:               base, project-config
+  hs-source-dirs:       app
+  main-is:              plutus-example.hs
+  ghc-options:          -threaded -rtsopts "-with-rtsopts=-T"
+
+  build-depends:        cardano-api
+                      , directory
+                      , filepath
+                      , plutus-example
+
+
+executable create-script-context
+  import:               base, project-config
+  hs-source-dirs:       app
+  main-is:              create-script-context.hs
+  ghc-options:          -threaded -rtsopts "-with-rtsopts=-T"
+
+  build-depends:        bytestring
+                      , cardano-api
+                      , optparse-applicative
+                      , plutus-example
+                      , transformers
+
+test-suite plutus-example-test
+  import:               base, project-config
+
+  hs-source-dirs:       test
+  main-is:              plutus-example-test.hs
+  type:                 exitcode-stdio-1.0
+
+  build-depends:        cardano-prelude
+                      , aeson
+                      , cardano-api
+                      , cardano-api:gen
+                      , cardano-ledger-alonzo
+                      , cardano-ledger-core
+                      , cardano-ledger-shelley
+                      , cardano-testnet
+                      , containers
+                      , directory
+                      , filepath
+                      , hedgehog
+                      , hedgehog-extras
+                      , plutus-example
+                      , plutus-ledger
+                      , plutus-ledger-api
+                      , tasty
+                      , tasty-hedgehog
+                      , text
+                      , unordered-containers
+
+
+  other-modules:
+                        -- TODO: Re-enable when cardano-node is bumped to the latest master
```

Comment:
Is this TODO relevant?

Reply:
Yeah, `plutus-apps` is pinned to a cardano-node commit in December